src/main/scala/dal/table/currencyTable.scala

```diff
@@ -0,0 +1,13 @@
+package dal.table
+
+import sqlest._
+
+class CurrencyTable(alias: Option[String]) extends Table(currency) {
+
+  val currencyCode = column[String]("currency_code")
+  val symbol = column[String]("symbol")
+  val description = column[String]("description")
+  val active = column[Boolean]("active")
```

Comment:
For Booleans, you may need to add an extra mapping function on the end. Here's an example from AS:

`val isIsa = column[Boolean]("is_isa")(BooleanYNColumnType)`